aiodocker/docker.py

```diff
@@ -42,7 +42,23 @@ def __init__(self,
                  connector=None,
                  session=None,
                  ssl_context=None,
-                 api_version='v1.26'):
+                 api_version='v1.26',
+                 loop=None):
```

Comment:
Using implicit loop is preferable technique starting from python 3.5.3
aiodocker is a new library, we don't need to pass loop everywhere.